help/configure-where-you-land.md

```diff
@@ -0,0 +1,39 @@
+# Configure where you land in message feeds
+
+When you go to a message feed in the Zulip web or desktop app, you'll generally
+land at your oldest unread message in that view.
+[Searches](/help/search-for-messages) and [starred
+messages](/help/star-a-message) put you at the newest message instead.
+
+
+In the mobile app, you can choose where message feeds will open:
+
+* **First unread message**: Recommended if you like to clear your inbox of
+  unread messages.
+* **First unread message in single conversations, newest message elsewhere**: In
+  Zulip, a **conversation** is a [direct message](/help/direct-messages) thread
+  (one-on-one or with a group), or a [topic in a
+  channel](/help/introduction-to-topics). This option works well if you have old
+  unread messages, and don't want to navigate there in your [Combined
+  feed](/help/combined-feed) and [channel feeds](/help/channel-feed).
+* **Newest message**: You may be used to this behavior from other chat applications.
```

Comment:
This might benefit from a tip to consider also setting the "Mark messages as read on scroll" to "never".

(If you choose this option but leave the other at its default of "In conversation views", then when you visit a conversation view with more than a screenful of history, you'll go to the end and immediately start marking those messages as read, while other messages remain unread above.)